src/main/java/com/andersenlab/lecture_6/bus_ticket/BusTicket.java

```diff
@@ -0,0 +1,33 @@
+package com.andersenlab.lecture_6.bus_ticket;
+
+import lombok.Data;
+
+import java.time.LocalDate;
+
+@Data
+public class BusTicket {
+
+    private int id;
+
+    private TicketType ticketType;
+
+    private LocalDate startDate;
+
+    private double price;
```

Comment:
Using double for money is considered as bad practice due to precision issues that can lead to rounding errors.